docs/content/latest/yugabyte-platform/install-yugabyte-platform/uninstall-software.md

```diff
@@ -12,63 +12,72 @@ isTocNested: true
 showAsideToc: true
 ---
 
-To uninstall Yugabyte Platform, follow the steps for Docker or Kubernetes environments.
+<!-- 
+
+You can uninstall Yugabyte Platform in the Kubernetes environments.
 
 ## Uninstall in Docker environments
 
-1. Stop and remove the Yugabyte Platform on Replicated first.
+You can stop and remove Yugabyte Platform on Replicated, as follows:
+
+1. Execute the following command to gain access to applications installed on Replicated:
 
     ```sh
-    $ /usr/local/bin/replicated apps
+    /usr/local/bin/replicated apps
     ```
 
-2. Replace <appid> with the application ID of Yugabyte Platform from the command above.
+2. To stop Yugabyte Platform, execute the following command, replacing *appid* with the application ID of Yugabyte Platform obtained from the preceding step:
 
     ```sh
-    $ /usr/local/bin/replicated app <appid> stop
+    /usr/local/bin/replicated app <appid> stop
     ```
 
-3. Remove the Yugabyte Platform application.
+THE rm COMMAND IN STEP 3 DOESN'T WORK, AS PER DEV. THIS IS WHY THIS WHOLE SECTION IS BEING REMOVED FOR NOW
+
+3. Remove Yugabyte Platform, as follows:
 
     ```sh
-    $ /usr/local/bin/replicated app <appid> rm
+    /usr/local/bin/replicated app <appid> rm
     ```
 
-4. Remove all Yugabyte Platform containers.
+2. Remove all Yugabyte Platform containers, as follows:
 
     ```sh
-    $ docker images | grep "yuga" | awk '{print $3}' | xargs docker rmi -f
+    sudo docker images | grep "yuga" | awk '{print $3}' | xargs docker rmi -f
     ```
 
-5. Delete the mapped directory.
+3. Delete the mapped directory, as follows:
 
     ```sh
-    $ rm -rf /opt/yugabyte
+    sudo rm -rf /opt/yugabyte
     ```
 
-6. Uninstall Replicated by following instructions documented in [Removing Replicated](https://help.replicated.com/docs/native/customer-installations/installing-via-script/#removing-replicated).
+6. Uninstall Replicated by following instructions provided in [Removing Replicated](https://help.replicated.com/docs/native/customer-installations/installing-via-script/#removing-replicated).
 
 ## Uninstall in Kubernetes environments
+-->
+
+You can uninstall Yugabyte Platform in Kubernetes, as follows:
 
-1. To remove the Yugabyte Platform, run the `helm delete` command:
+1. To remove the Yugabyte Platform, execute the following helm command:
 
     ```sh
-    $ helm del yw-test -n yw-test
+    helm del yw-test -n yw-test
     ```
 
-    A message displays that the Yugabyte Platform release and the namespace is deleted.
+    <br>You should see a message similar to the following, notifying you that the Yugabyte Platform release and the namespace is deleted:
```

Comment:
namespace is deleted? but following step is to delete namespace

Reply:
Fixed